client/components/team/imports/team-helpers.js

```diff
@@ -3,28 +3,34 @@ import { withTracker } from 'meteor/react-meteor-data';
 
 export const DIVISION_TYPES = [
   {
-    text: 'WWU Student',
-    value: 'wwu-student',
+    text: 'WWU Students',
+    value: 'wwu-students',
```

Comment:
Is there any logic in this file or elsewhere that will get confused by `students` instead of `student`?

Reply:
It should be fine, there are only a couple occurrences of that string in the whole repository, all of which I changed. I just thought plural might make more sense!